app/src/main/java/com/github/gotify/service/WebSocketService.java

```diff
@@ -297,13 +304,19 @@ private void showNotification(
                 .setTicker(getString(R.string.app_name) + " - " + title)
                 .setGroup(NotificationSupport.Group.MESSAGES)
                 .setContentTitle(title)
-                .setContentText(message)
-                .setStyle(new NotificationCompat.BigTextStyle().bigText(message))
                 .setDefaults(Notification.DEFAULT_LIGHTS | Notification.DEFAULT_SOUND)
                 .setLights(Color.CYAN, 1000, 5000)
                 .setColor(ContextCompat.getColor(getApplicationContext(), R.color.colorPrimary))
                 .setContentIntent(contentIntent);
 
+        CharSequence formattedMessage = message;
+        if (Extras.useMarkdown(extras)) {
+            formattedMessage = markwon.toMarkdown(message);
+            message = formattedMessage.toString();
```

Comment:
Do you know if this is also supported for older android versions?

Reply:
No, I'll have to check it.

Reply:
Yes it's working. Tested on Android 5.0/Lollipop/API 21. Looks even a little better, because the different heading sizes aren't ignored and the last two lines aren't truncated xD
![Screenshot_1628189638](https://user-images.githubusercontent.com/87871070/128406541-8c52cfc2-4518-4a1f-b6ad-a24a115e728a.png)

Reply:
Excellent, the code looks fine. I'll test this on the weekend and then merge it into master.